SU-HSDO/suhumsci
# STN-372 : Traditional Animations

#  [STN-372](https://sparkbox.atlassian.net/browse/STN-372) READY FOR REVIEW

## Summary
Adjust the hover transitions and animations for the Traditional theme.

This PR refactors the `hb-transition` mixin to output transition duration and timing values based upon the theme.

The work in this PR has been reviewed by a designer (J) and browser tested. ✅

## Need Review By (Date)
5/20/2020

## Urgency
medium

## Steps to Test
1. Run `npm run test` and verify that all of our Sass tests pass.
2. Review the following items in the animations list below. Test both colorful and traditional themes and verify that the items below receive the correct animation timing and duration as declared in their respective theme animation variables.

### Animations
- [x] Vertical Linked Card
- [x] More link arrow icons in Cards and Postcards
- [x] Pills
- [x] Pagers
- [x] Images - in horizontal and structured cards, and WYSIWYG
- [x] Buttons
- [x] Exposed filters search choice - Go to the `Courses` pages, under `Quarter`, select `Fall`. This is the search choice. The transition is on hover.
- [x] Font Awesome Icons - check out linked icons in footer
- [x] Category links
- [x] Headings
- [x] Main content links for Traditional should not have the background-image upon hover
